corneish_zen_v2  build error

I just followed the steps in here: https://zmk.dev/docs/user-setup
in Github actions keep getting this error

**fatal error: layouts/foostan/corne.dtsi: No such file or directory**

```
Run west build -s zmk/app -d "/tmp/tmp.9zJXJAuEK3" -b "corneish_zen_v2_left"  -- -DZMK_CONFIG=/tmp/zmk-config/config  -DZMK_EXTRA_MODULES='/__w/zmk-config/zmk-config' 
+ west build -s zmk/app -d /tmp/tmp.9zJXJAuEK3 -b corneish_zen_v2_left -- -DZMK_CONFIG=/tmp/zmk-config/config -DZMK_EXTRA_MODULES=/__w/zmk-config/zmk-config
-- west build: generating a build system
Loading Zephyr default modules (Zephyr base).
-- Application: /tmp/zmk-config/zmk/app
-- CMake version: 3.30.0
-- Found Python3: /usr/bin/python3 (found suitable version "3.[12](https://github.com/Mograbi/zmk-config/actions/runs/11163468991/job/31030512387#step:11:13).3", minimum required is "3.8") found components: Interpreter
-- Cache files will be written to: /tmp/zmk-config/zephyr/.cache
-- Zephyr version: 3.5.0 (/tmp/zmk-config/zephyr)
-- Found west (found suitable version "1.2.0", minimum required is "0.14.0")
-- ZMK Config directory: /tmp/zmk-config/config
-- ZMK Config Kconfig: /tmp/zmk-config/config/corneish_zen.conf
-- Using keymap file: /tmp/zmk-config/config/corneish_zen.keymap
-- Using keymap file: /tmp/zmk-config/config/corneish_zen.keymap
-- Board: corneish_zen_v2_left
-- ZEPHYR_TOOLCHAIN_VARIANT not set, trying to locate Zephyr SDK
-- Found host-tools: zephyr 0.16.3 (/opt/zephyr-sdk-0.16.3)
-- Found toolchain: zephyr 0.16.3 (/opt/zephyr-sdk-0.16.3)
-- Found Dtc: /opt/zephyr-sdk-0.16.3/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.6.0", minimum required is "1.4.6")
-- Found BOARD.dts: /tmp/zmk-config/zmk/app/boards/arm/corneish_zen/corneish_zen_v2_left.dts
-- Found devicetree overlay: /tmp/zmk-config/config/corneish_zen.keymap
In file included from /tmp/zmk-config/zmk/app/boards/arm/corneish_zen/corneish_zen_v2_left.dts:8,
                 from <command-line>:
/tmp/zmk-config/zmk/app/boards/arm/corneish_zen/corneish_zen.dtsi:[13](https://github.com/Mograbi/zmk-config/actions/runs/11163468991/job/31030512387#step:11:14):10: fatal error: layouts/foostan/corne.dtsi: No such file or directory
   13 | #include <layouts/foostan/corne.dtsi>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
CMake Error at /tmp/zmk-config/zephyr/cmake/modules/extensions.cmake:3885 (message):
  failed to preprocess devicetree files (error code 1):
  /tmp/zmk-config/zmk/app/boards/arm/corneish_zen/corneish_zen_v2_left.dts;/tmp/zmk-config/config/corneish_zen.keymap
Call Stack (most recent call first):
  /tmp/zmk-config/zephyr/cmake/modules/dts.cmake:[22](https://github.com/Mograbi/zmk-config/actions/runs/11163468991/job/31030512387#step:11:23)6 (zephyr_dt_preprocess)
  /tmp/zmk-config/zephyr/cmake/modules/zephyr_default.cmake:129 (include)
  /tmp/zmk-config/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:66 (include)
  /tmp/zmk-config/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:92 (include_boilerplate)
  CMakeLists.txt:8 (find_package)
-- Configuring incomplete, errors occurred!
FATAL ERROR: command exited with status 1: /usr/local/bin/cmake -DWEST_PYTHON=/usr/bin/python3 -B/tmp/tmp.9zJXJAuEK3 -GNinja -DBOARD=corneish_zen_v2_left -DZMK_CONFIG=/tmp/zmk-config/config -DZMK_EXTRA_MODULES=/__w/zmk-config/zmk-config -S/tmp/zmk-config/zmk/app
Error: Process completed with exit code 1.
```

--- a/app/boards/arm/corneish_zen/corneish_zen.dtsi
+++ b/app/boards/arm/corneish_zen/corneish_zen.dtsi
@@ -10,7 +10,8 @@
 
 #include <dt-bindings/zmk/matrix_transform.h>
 
-#include <layouts/foostan/corne.dtsi>
+#include <layouts/foostan/corne/5column.dtsi>
+#include <layouts/foostan/corne/6column.dtsi>
 
 &foostan_corne_6col_layout {
     transform = <&default_transform>;

